MediaPushPlugin/src/main/java/io/antmedia/plugin/MediaPushPlugin.java

```diff
@@ -375,13 +396,71 @@ public String getPublisherHTMLURL(String websocketUrl) throws InvalidArgumentExc
 		return publisherHtmlURL;
 	}
 
+	public boolean isValidIP(String ip) {
+		if (ip == null)
+			return false;
+		return ip.matches("^((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)\\.){3}(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)$");
+	}
+
+	public boolean forwardMediaPushStopRequest(String streamId, String ip) {
```

Comment:
I think it's a MUST to have integration test for this code. We may create another issue or increase the story point of this issue